models/repo.go

```diff
@@ -264,6 +264,11 @@ func (repo *Repository) HTMLURL() string {
 	return setting.AppURL + repo.FullName()
 }
 
+// APIURL returns the repository API URL
+func (repo *Repository) APIURL() string {
+	return strings.TrimRight(setting.AppURL, "/") + "/" + path.Join("api/v1/repos", repo.FullName())
```

Comment:
@bkcsoft Please see the result from your url

```
https:/try.gitea.io/api/v1/repos/user01/repo01
```

Wrong: `https:/`
Correct: `https://`